packages/mantine/src/theme/Theme.tsx

```diff
@@ -212,6 +215,8 @@ export const plasmaTheme: MantineThemeOverride = createTheme({
             },
             vars: NotificationVars,
         }),
+        PlasmaTable: Table.extend({classNames: TableClasses}),
+        RowLayout: RowLayout.extend({classNames: RowLayoutClasses}),
```

Comment:
Should we be using this here?

```suggestion
        RowLayout: Table.Layouts.Rows.extend({classNames: RowLayoutClasses}),
```